src/scripts/analyze.sh

```diff
@@ -126,6 +127,9 @@ echo -n "Starting a static analysis ..."
 $CLI_LOCATION --directory "${PROJECT_ROOT}" -t true -o "${OUTPUT_FILE}" -f sarif || exit 1
 echo "done"
 
+# navigate to project root, so the datadog-ci command can access the git info
+cd "$PROJECT_ROOT" || exit 1
```

Comment:
Is the `|| exit 1` required? Could the `cd` command fails happen?

Reply:
The user can always attempt to `unset PROJECT_ROOT` so it's unlikely but it may happen.

On the other hand, we may need it for `git config --global --add safe.directory "${PROJECT_ROOT}"` - do we want to fail if we fail to add the `safe.directory` to git? It may be a silent error if it fails.

Reply:
In conclusion, we want this now?

`git config --global --add safe.directory "${PROJECT_ROOT}" || exit 1`

Reply:
yes